src/System.ComponentModel.Annotations/tests/ScaffoldColumnAttributeTests.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Xunit;
+
+namespace System.ComponentModel.DataAnnotations
+{
+    public class ScaffoldColumnAttributeTests
+    {
+        [Fact]
+        public static void Can_construct_and_get_Scaffold()
+        {
+            var attribute = new ScaffoldColumnAttribute(true);
+            Assert.Equal(true, attribute.Scaffold);
```

Comment:
Consider adding the case for ```false``` as well, since as-is the test could pass if the implementation were erroneously hard-coded to always return true.

The assert could also just be ```Assert.True(attribute.Scaffold)```, though if you were to do this using a [Theory] and [InlineData] to handle the multiple cases, you'd want to stick with Equal, e.g.
```C#
[Theory]
[InlineData(true)]
[InlineData(false)]
public void Can_construct_and_get_Scaffold(bool value)
{
    var attribute = new ScaffoldColumnAttribute(value);
    Assert.Equal(value, attribute.Scaffold);
}
```
or something like that.